modules/compute/vm-instance/README.md

```diff
@@ -122,6 +122,23 @@ Use the following settings for spread placement:
 More information on GPU support in `vm-instance` and other HPC Toolkit modules
 can be found at [docs/gpu-support.md](../../../docs/gpu-support.md)
 
+## Lifecycle
+
+The `vm-instance` module will be replaced when the `instance_image` variable is
+changed and `terraform apply` is run on  the deployment group folder or
+`ghpc deploy` is run. However, it will not be automatically replaced if a new
+image is created in a family.
+
+To selectively replace the vm-instance(s), consider running terraform
+`apply -replace` such as:
+
+```shell
+terraform state list
+# search for the module ID and resource
+# search for the module ID and resource
+terraform apply -replace="address"
```

Comment:
I know you're dpulicating my exact text, so sorry for change request. I suggest having a comment above this line with

```text
# See https://developer.hashicorp.com/terraform/cli/commands/plan#replace-address for precise syntax
terraform apply -replace=ADDRESS
```

Reply:
replace # by > and added it in the text, rather than the code.